README.md

```diff
@@ -1,236 +1,22 @@
-
-The following sections provide you details on how to use the Slack connector.
-
-- [Feature Overview](#feature-overview)
-- [Quickstart](#quickstart)
-- [Samples](#samples)
-
-## Feature Overview
-
-1. Conducting messaging-related operations. For example, post messages on slack, delete messages, 
-send attachments, etc.
-2. Executing `conversations/channels`-related operations. For example, create conversations,
-join a conversation, add users to a conversation, archive/unarchive conversations, etc.
-3. Conducting `users/user groups`-related operations. For example, get user information etc.
-4. Performing file-related operations in Slack. For example, upload files, delete files, get file information, etc.
-
-# Quickstart
-
-### Pull the Module
-Execute the below command to pull the Slack module from Ballerina Central:
-```ballerina
-$ ballerina pull ballerinax/slack
-```
-
-### Slack Client Sample
-The Slack Client Connector can be used to interact with the Slack Web API.
-
-```ballerina
-import ballerina/log;
-import ballerinax/slack;
-import ballerina/os;
-
-slack:Configuration slackConfig = {
-    bearerTokenConfig: {
-        token: os:getEnv("SLACK_TOKEN")
-    }
-};
-
-public function main() returns error? {
-    slack:Client slackClient = check new(slackConfig);
-
-    slack:Message messageParams = {
-        channelName: "channelName",
-        text: "Hello"
-    };
-
-    // Post a message to a channel.
-    var postResponse = slackClient->postMessage(messageParams);
-    if (postResponse is string) {
-        log:printInfo("Message sent");
-    } else {
-        log:printError(postResponse.toString());
-    }
-
-    // List all the conversations.
-    var listConvResponse = slackClient->listConversations();
-    if (listConvResponse is error) {
-        log:printError(listConvResponse.toString());
-    } else {
-        log:printInfo(listConvResponse.toString());
-    }
-
-    // Upload a file to a channel.
-    var fileResponse = slackClient->uploadFile("filePath", "channelName");
-    if (fileResponse is error) {
-        log:printError(fileResponse.toString());
-    } else {
-        log:printInfo("Uploaded file " + fileResponse.id);
-    }
-
-    // Get user information.
-    var userResponse = slackClient->getUserInfo("userName");
-    if (userResponse is error) {
-        log:printError(userResponse.toString());
-    } else {
-        log:printInfo("Found user information of the user ", userResponse.name);
-    }
-}
-```
-## Please check the [Samples directory](https://github.com/ballerina-platform/module-ballerinax-slack/tree/master/samples) for more examples.
-
-
-## Module Overview - `ballerinax/slack.'listener`
-
-The `ballerinax/slack.'listener` module provides a Listener to grasp event triggers from your Slack App. This functionality is provided by [Slack Event API](https://api.slack.com/apis/connections/events-api). 
+This connector allows you to access the Slack Web API and Slack Events API through Ballerina.
```

Comment:
It provides the capability to query information and perform some actions in a Slack workspace. It also allows you to listen to Slack Events. -> some actions? shall we specify what they are?

Reply:
That is a big list. That list is available in the api docs. In choreo, those will be listed in a dropdown.
I think we don't need to repeat them here.

Reply:
Shall we specify a few and say etc.